### PR TITLE
C353633 Verify that the in-app bulk edit preview  contains affected r…

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C353633](https://foliotest.testrail.io/index.php?/cases/view/353633)

[Allure Report ](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5546/allure)

image

### CODE REVIEW

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    after('delete test data', () => {
+      InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(item.firstBarcode);
+      Users.deleteViaApi(user.userId);
+      FileManager.deleteFile(`cypress/fixtures/${itemBarcodesFileName}`);
+      FileManager.deleteFileFromDownloadsByMask(previewFileName);
+    });
+
+    it(
+      'C353633 Verify that the in-app bulk edit preview contains affected records (firebird) (TaaS)',
+      { tags: [TestTypes.smoke, DevTeams.firebird] },
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js:61]: is this a smoke test?

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import FileManager from '../../../support/utils/fileManager';
+import Users from '../../../support/fragments/users/users';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import InventoryItems from '../../../support/fragments/inventory/item/inventoryItems';
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    let user;
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js:15]: better to declare these outside of describe block

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  instanceName: `instanceName-${getRandomPostfix()}`,
+  firstBarcode: barcode,
+  secondBarcode: `secondBarcode_${barcode}`,
+  annexId: '53cf956f-c1df-410b-8bea-27f712cca7c0',
+};
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    before('create test data', () => {
+      cy.getAdminToken();
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js:27]: cy.createTempUser already has       cy.getAdminToken();
should work without it

**Review by manowarqqq — APPROVED** [review]

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js (hunk 3 above)
Comment (KarapetyanSofia) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-affected-records.cy.js]: I already fixed it!

**Review by manowarqqq — APPROVED** [review]

**Review by nayimovag — APPROVED** [review]

**Pull request merged**